### PR TITLE
A typo in HPA Algorithm Details section

### PR DESCRIPTION
**Pull request by pipo02mix**
The HPA task doc page has a typo (IMO) in the description of how the algorithm works

### CODE REVIEW

**Comment by kbhawkey** [comment]
Hello @pipo02mix .
Thanks for this pull request.

Page preview:
https://deploy-preview-20802--kubernetes-io-master-staging.netlify.app/docs/tasks/run-application/horizontal-pod-autoscale/#algorithm-details

**Comment by kbhawkey** [comment]
@pipo02mix ,
Indeed, I find the sentence unclear.

Ping @kubernetes/autoscaler-reviewers for review. Thanks!

**Comment by bskiba** [comment]
@josephburnett

**Comment by kbhawkey** [comment]
@josephburnett , Do you have time to review this PR?

File: content/en/docs/tasks/run-application/horizontal-pod-autoscale.md [hunk 1]
```diff
@@ -123,7 +123,7 @@ Due to technical constraints, the HorizontalPodAutoscaler controller
 cannot exactly determine the first time a pod becomes ready when
 determining whether to set aside certain CPU metrics. Instead, it
 considers a Pod "not yet ready" if it's unready and transitioned to
-unready within a short, configurable window of time since it started.
+ready within a short, configurable window of time since it started.
```
Comment (sftim) [inline content/en/docs/tasks/run-application/horizontal-pod-autoscale.md:126]: I'm afraid I think the original wording is correct @pipo02mix 

I'd be happy to review pull requests that aim to clarify the existing behaviour, as the documentation is hard to follow here.

Comment (pipo02mix) [reply content/en/docs/tasks/run-application/horizontal-pod-autoscale.md:126]: I will try to come with a simpler phrase :D
